facades.md

```diff
@@ -0,0 +1,58 @@
+# Adding a new facade to third-party-web
+
+Lighthouse 7.0 includes a new audit to flag third-party 
+embeds (link coming soon) that can be lazy loaded with a facade. The 
+documentation lists recommended facades. If you'd like to add a facade to the 
+list of recommendations, please read the guidance below.
+
+## Terminology
+
+* **third-party product:** a third-party resource that is included on a page.
+    * For example: a YouTube embed iframe.
+* **facade**: a frontend component/library/project of HTML/CSS/JS/etc that 
```

Comment:
I think we can shorten this to just "a frontend component of...". A facade bing a library or project is pretty much implied.

Reply:
```suggestion
* **facade**: a frontend component of HTML/CSS/JS/etc that 
```